app/searchcriteria.py

```diff
@@ -79,6 +79,32 @@ async def clear_stored_search_criteria(session, microservice_name=''):
       del session[key_to_clear]
 
 
+def device_sent_dropdown(dropdown_value):
+  if dropdown_value == '':
+    dropdown_value = "blank"
+  dropdown_options = [
+      {
+          'value': 'blank',
+          'text': 'Select a value',
+          "disabled": True
```

Comment:
Style note- these should be single quotes.

Using a string 'blank' as the none-value seems a bit odd, when we should perhaps be using Python's None value for this. `dropdown_value=None` for a default value, no need to set it to `'blank'` if it's the empty string, with the blank value set as `'value': None`.